test/e2e/storage/csi_volumes.go

```diff
@@ -361,7 +361,17 @@ func getTestPod(t testsuites.StorageClassTest, claim *v1.PersistentVolumeClaim,
 
 func testCSIDriverUpdate(f *framework.Framework, driver drivers.TestDriver) {
 	// NOTE: this test assumes that all CSI drivers in csiTestDrivers deploy exactly one DaemonSet.
-	It("should work after update", func() {
+	var namespace string
+	testName := "should work after update"
+	if driver.GetDriverInfo().FeatureTag == drivers.GcePDExternalFeatureTag {
```

Comment:
this would have to distinguish between "actual namespace" and "use test framework namespace", and probably go in driver info, something like
```
type DriverInfo struct{
...
DriverInstalledPerTestNamespace bool
DriverInstalledNamespace string
...
}
```
?? 
Is there a better way to represent such a configuration

Reply:
Maybe just the string is enough.  If it's empty, then assume the test namespace.